torch/csrc/autograd/python_engine.cpp

```diff
@@ -97,13 +97,33 @@ void compute_partial_exec_callbacks(const function_list& roots,
     }
   }
 
-  // Prevent expantion for functions in {all_vertices} \ {needed}
+  // Prevent expansion for functions in {all_vertices} \ {needed}
   for (auto fn : all_functions) {
     if (needed.count(fn) > 0) continue;
     map.emplace(fn, abort_callback);
   }
 }
 
+// Implementation of torch._C._EngineBase.run_backward
+// Arguments:
+//  variables (tuple of Variable):
+//      Variables of which the derivative will be computed.
+//      These must not be volatile.
+//  grad_variables (tuple of Variable or None):
+//      Gradients with respect to each element of corresponding
+//      variables.  Has same length as variables.  See _make_grads
+//      how these are defaulted when the user doesn't specify them.
+//  keep_graph (boolean):
+//      If false, the graph used to compute the grad will be freed.
+//  inputs (tuple of Variable)
+//      Inputs w.r.t. which the gradient will be returned.  This
+//      function will return a tuple containing an entry for each
+//      input variable.  Non-mentioned leaf variables will have
+//      gradients accumulated to grad, unless only_inputs below
+//      is True.
+//  only_inputs (boolean)
+//      If false, any inputs not specified in inputs will have their
+//      gradients accumulated to grad.
```

Comment:
I don't it's a good idea to replicate Python docs here. You can always look them up, and they'll quickly get out of date